executor/admin_test.go

```diff
@@ -1322,144 +1322,12 @@ func TestCheckFailReport(t *testing.T) {
 	}()
 }
 
-func (s *testSuiteJoinSerial) TestAdminCheckTableFailed(t *testing.T) {
```

Comment:
Why removing this test case?

Reply:
It's been moved to admin_serial_test.go. The deleted part is mistakenly reserved from a merge.